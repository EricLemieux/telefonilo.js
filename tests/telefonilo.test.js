const Telefonilo = require('../src/telefonilo')();

describe('Test helper functions', () => {
    describe('isMobile()', () => {
        it('should check if the user is on Mobile', () => {
            expect(true).toBeFalsy();
        });
    
        it('should check if the user is NOT on Mobile', () => {
            expect(true).toBeFalsy();
        });
    });

    describe('isValidEncryptionFunction(_function)', () => {
        it('should return false if it`s not a valid function', () => {
            expect(true).toBeFalsy();
        });

        it('should return false if it returns the same parameters', () => {
            expect(true).toBeFalsy();
        });
    });
});

describe('Test Telephonilo', () => {
    it('should be invoked without parameters', () => {
        expect(true).toBeFalsy();
    });

    describe('Accept CSS selectors as parameter', () => {
        it('should work with classed (`.class`) ', () => {
            expect(true).toBeFalsy();
        });

        it('should work with IDs (`#myID`) ', () => {
            expect(true).toBeFalsy();
        });

        // TODO: Correct me
        it('should work with data attributes (`data-attr`) ', () => {
            expect(true).toBeFalsy();
        });

    });
})