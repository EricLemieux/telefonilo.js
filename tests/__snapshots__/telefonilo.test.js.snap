// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Telefonilo should match the snapshot for the test html 1`] = `
<body>
  <div
    class="container"
  >
    <p>
      If you're on a mobile phone you should click on
      <span
        class="phone-num"
      >
        0123456789
      </span>
    </p>
  </div>
</body>
`;

exports[`Test Telefonilo should work for non-mobile devices 1`] = `
<body>
  <div
    class="container"
  >
    <p>
      If you're on a mobile phone you should click on
      <span
        class="phone-num"
      >
        0123456789
      </span>
    </p>
  </div>
</body>
`;
